web/client/components/contextcreator/ContextCreator.jsx

```diff
@@ -18,7 +18,8 @@ export default class ContextCreator extends React.Component {
         curStepId: PropTypes.string,
         newContext: PropTypes.object,
         onChangeAttribute: PropTypes.func,
-        onSave: PropTypes.func
+        onSave: PropTypes.func,
+        onSaveDestLocation: PropTypes.string
```

Comment:
hi, i would change th name of this prop since on...  usually indicates a function
```suggestion
        saveDestLocation: PropTypes.string
```